c7n/actions/network.py

```diff
@@ -214,16 +255,30 @@ def get_groups(self, resources):
             isolation_groups = self.resolve_group_names(
                 r, self._get_array('isolation-group'), resolved_groups)
 
+            add_by_tag_groups = []
+            for sg in tag_filtered_groups:
+                if sg['VpcId'] in r['VpcId']:
```

Comment:
Shouldn't this be equality and instead of tracking in a new variable why not add to extant add groups car

Reply:
@kapilt I changed to the equality operator (==), and add tagged security groups to `add_groups` instead of using a new variable